docker/1.14/Dockerfile.eia

```diff
@@ -1,6 +1,5 @@
 FROM ubuntu:16.04
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
-LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
```

Comment:
1. not clear what this change has to do with ping
2. does this mean we are removing the mme feature from this version? that's a breaking change.